examples/SpringBootHikariExample/README.md

```diff
@@ -0,0 +1,145 @@
+# Tutorial: Getting started with Spring Boot, Hikari, and the AWS Advanced JDBC Wrapper Driver
+
+In this tutorial, you will set up a Spring Boot application using Hikari and the AWS Advanced JDBC Wrapper Driver.
+
+> Note: this tutorial was written using the following technologies:
+>    - Spring Boot 3.1.0
+>    - AWS Advanced JDBC Wrapper 2.1.2 
+>    - Postgresql 42.5.4
+>    - Kotlin
+>    - Java 8
+
+
+## Step 1: Create a Gradle Project
+Create a Gradle Project with the following project hierarchy:
+```
+├───gradle
+│   └───wrapper
+│       ├───gradle-wrapper.jar
+│       └───gradle-wrapper.properties   
+├───build.gradle.kts
+├───gradlew
+└───src
+    └───main
+        ├───java
+        │   └───software
+        │       └───amazon
+        │           └───SpringBootHikariExampleApplication.java
+        └───resources
+            └───application.yml
+```
+When creating the `SpringBootHikariExampleApplication.java` class, fill add the following code to it.
+
+```
+package software.amazon.SpringBootHikariExample;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringBootHikariExampleApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(SpringBootHikariExampleApplication.class, args);
+	}
+
+}
+```
+You may also use the Spring Initializr to create the boilerplate code:
+1. Go to https://start.spring.io/
+2. Select the Maven project and version 2.7.9 of the Spring Boot.
+3. Select Java version 8.
+4. Click Dependencies and select the following:
+    - Spring Web
+    - Spring Data JDBC
+    - PostgreSQL Driver
+
+## Step 2: Add the required Gradle Dependencies
+
+In the `build.gradle.kts` file, add the following dependencies.
+
+```
+dependencies {
+	implementation("org.springframework.boot:spring-boot-starter-data-jdbc")
+	implementation("org.springframework.boot:spring-boot-starter-web")
+	implementation(project(":aws-advanced-jdbc-wrapper"))
+	implementation("org.postgresql:postgresql:42.5.1")
+}
+```
+
+## Step 3: Configure the Datasource
+
+In the `application.yml` file, configure Hikari and AWS Advanced JDBC Wrapper Driver as its driver.
+
+```
+spring:
+  datasource:
+    url: jdbc:aws-wrapper:postgresql://database-endpoint-url:5432/database-name
```

Comment:
To keep things consistent, can we use the sample values for endpoints and credentials that we have in the other example files?